packages/prepare.js

```diff
@@ -4,38 +4,57 @@ const util = require('util');
 const execFile = util.promisify(require('child_process').execFile);
 
 const rootDir = path.resolve(__dirname, '..');
-const srcDir = path.join(rootDir, 'ed/idl');
-const patchDir = path.join(rootDir, 'ed/idlpatches');
-const dstDir = path.join(rootDir, 'packages/idl');
+
+const packages = [
+  {
+    name: 'idl',
+    srcDir: path.join(rootDir, 'ed/idl'),
+    dstDir: path.join(rootDir, 'packages/idl'),
+    patchDir: path.join(rootDir, 'ed/idlpatches'),
+    fileExt: 'idl'
+  },
+  {
+    name: 'css',
+    srcDir: path.join(rootDir, 'ed/css'),
+    dstDir: path.join(rootDir, 'packages/css'),
+    patchDir: path.join(rootDir, 'ed/csspatches'),
+    fileExt: 'json'
+  },
+];
+
 
 async function main() {
-  // rm dstDir/*.idl
-  const dstFiles = await fs.readdir(dstDir);
-  for (const file of dstFiles) {
-    if (file.endsWith('.idl')) {
-      await fs.unlink(path.join(dstDir, file));
+  for (const { name, srcDir, dstDir, patchDir, fileExt } of packages) {
+    // rm dstDir/*.${fileExt}
+    const dstFiles = await fs.readdir(dstDir);
+    for (const file of dstFiles) {
+      if (file.endsWith('.' + fileExt) && (file !== 'package.json')) {
```

Comment:
```suggestion
      if (file.endsWith(`.${fileExt}`) && file !== 'package.json') {
```

Although it would actually be simpler if `fileExt` included the leading period.